src/OpenLoco/Core/Variant.hpp

```diff
@@ -7,15 +7,11 @@
 #if !defined(__APPLE__)
 
 #include <variant>
-namespace stdx
-{
-    using std::variant;
-    using std::visit;
-}
+
 #else
 
 #include "../../Thirdparty/variant.hpp"
-namespace stdx
+namespace std
```

Comment:
You can't add things to the std namespace.

https://stackoverflow.com/questions/41062294/c-when-is-it-ok-to-extend-the-std-namespace

Reply:
I know this rule but this is less of an addition rather a substitution for something missing since its guarded with a macro. And yes normally you get a name clash if you wanted to implement a vector in std namespace. The change is to make this actually agnostic. Anyway I don't mind dropping it.